src/query/sql/src/planner/optimizer/util.rs

```diff
@@ -12,9 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use common_datavalues::type_coercion::compare_coercion;
```

Comment:
Prefer split(rename) this `util.rs` into separate files, otherwise, it will be bigger and bigger.